frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -592,3 +602,17 @@ func buildCvdCommand(execContext ExecContext,
 	finalArgs = append(finalArgs, args...)
 	return execContext("sudo", finalArgs...)
 }
+
+// Validates whether the current host is valid to run CVDs.
+type HostValidator struct {
+	ExecContext ExecContext
+}
+
+func (v *HostValidator) Validate() error {
+	cmd := v.ExecContext("grep", "-cw", "vmx", "/proc/cpuinfo")
```

Comment:
I'm worried about the forward compatibility of this check. For example, is this still correct if the host is ARM based instead of intel?
Cuttlefish only requires (and checks for) the availability of kvm. Could we do the same?

Reply:
good catch! did as suggested.